docs/content/configuration/handling-host-and-listener-collisions.md

```diff
@@ -2,7 +2,7 @@
 docs: DOCS-590
 doctypes:
 - ''
-title: Handling Host and Listener Collisions
+title: Host and Listener collisions
```

Comment:
I think we should keep the original heading, new on comes off a bit negative in the context

Reply:
I didn't realise I hadn't done a full sweep of this doc: updating it now.

I stand by the page title, but I think this is more of a reference page than a how-to, as it largely explains the mechanisms of how it works: it's not particularly instructional, so I don't think it makes sense under "Configuration".

Reply:
okay in that case we should rename the md file too for consistency